docs: Update Uno.Sdk page with latest changes (#20179)
unoplatform/uno: doc/articles/concepts/overview/why-uno-platform.md

Your Uno Platform WebAssembly app can run as a Progressive Web App. This means users can install it and run as a normal application on their device. In addition, this gives you even more customization options and additional PWA-specific APIs like badge notifications. You can install the [NuGet Package Explorer](https://nuget.info/) PWA like a local app!

> ## WebView across platforms - included at no extra cost

Reviewer: no need for 'extra' - as that could impy there is initial cost. Just 'no cost'

> ## WebView across platforms - included at no extra cost

Reviewer: Why are we not calling out specific of WebView2?

When you need to display web content within your cross-platform apps, you can use our `WebView2` control. Not only will it easily handle rendering of both remote and in-app HTML content, but it will also allow you to implement bi-directional JavaScript communication without any hassle. It is included in Uno Platform at no cost. The WebView2 control is available on all platforms.

## Media Player across platforms - included at no cost

> Uno Platform provides the `MediaPlayerElement` control which allows your app to play media files in your app across all platforms. The Media Player control is fully supported on all platforms, including Linux through libVLC integration and it is included with Uno Platform for free.

Reviewer 2: ```suggestion
Uno Platform provides the `MediaPlayerElement` control, which allows your app to play media files in your app across all platforms. The Media Player control is fully supported on all platforms, including Linux through libVLC integration, and it is included in Uno Platform at no cost.
```
